infrastructure/template/root/.github/workflows/test-and-pre-release.yml

```diff
@@ -77,3 +77,53 @@ jobs:
           $env:QGIS_PLUGIN_IN_CI=1
           python-qgis-ltr.bat -m pip install -q pytest
           python-qgis-ltr.bat -m pytest -v
+
+  pre-release:
+    name: "Pre Release"
+    runs-on: "ubuntu-latest"
+    needs: [linux_tests, windows_tests]
+
+    steps:
+      - uses: hmarr/debug-action@v2
```

Comment:
This should probably be removed in the production.

Reply:
Well, depends on how much time you want to use debugging it every time you want to change it :D

Reply:
In general, the lack of verbosity IMO is a problem for GitHub Actions in general, this one at least provides some basic output on every run.